### DISCUSSION TITLE
:heart: Enable Emoji Rendering in Title of Pull Request

### DISCUSSION THREAD

**Discussion by sangwoo-joh** [community/community#4572] [Pull Requests] [answered]
Always thanks for providing such a great tool!

Currently, GitHub emoji (e.g. `:+1:` as :+1: ) is only rednered in the body of pull requests or issues and in commits.
As a developer who prefers [gitmoji](https://gitmoji.dev/) process, it would be really nice if emoji is rendered in the title, too.
Because when a pull request is squash & merged, that title is used as a commit message, and I can see the nice emojis in the commits. However, I only see bare text like `:sparkles: add new feature` in the title of the pull request.

Please support this feature!

**Comment by emmaviolet** [comment]
Just a note to say we're looking into this

**Comment by emmaviolet** [comment, marked as answer]
@sangwoo-joh this appears to be working for us. What are we missing?
<img width="310" alt="Screenshot 2021-09-09 at 20 39 31" src="https://user-images.githubusercontent.com/8298818/132751776-eb12ba01-9ddb-463d-9e42-8c0bcd6ed359.png">

**Reply by sangwoo-joh** [reply]
@emmaviolet 
Thanks for your update! I've found that emoji rendering works in PR title!
Thank you!
